Automattic/wp-calypso
# TitleFormatEditor: Remove polyfills needed by draft-js

These polyfills are now provided by the environment via babel/preset-env and no longer need to be hard-coded in.


### Testing instructions
Pull up the title format editor in the SEO settings and make sure it still works in IE11.
